tests/DocumentationExamplesTest.php

```diff
@@ -925,6 +925,41 @@ public function testExample_55_58()
         $this->assertInventoryCount(0);
     }
 
+    public function testExample_59_61()
```

Comment:
In the event we get more generic examples later, we should probably call this `testChanceStreamExample_1_3()`.